src/main/java/javaQuotes/App.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package javaQuotes;
+
+import com.google.gson.Gson;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+public class App {
```

Comment:
This application should also pick one quote at random to spit out to the console.